mkosi/config.py

```diff
@@ -684,7 +685,13 @@ def from_namespace(cls, ns: argparse.Namespace) -> "MkosiArgs":
 
     def to_json(self, *, indent: Optional[int] = 4, sort_keys: bool = True) -> str:
         """Dump MKosiArgs as JSON string."""
-        return json.dumps(dataclasses.asdict(self), cls=MkosiJsonEncoder, indent=indent, sort_keys=sort_keys)
+        def key_transformer(k: str) -> str:
+            if (s := SETTINGS_LOOKUP_BY_DEST.get(k)) is not None:
+                return s.name
+            return "".join(p.capitalize() for p in k.split("_"))
```

Comment:
Does this make sense for `MkosiArgs`? It won't have anything in the settings

Reply:
Without the tests fail because of `AutoBump`

Reply:
I'm confused, we don't have an `AutoBump` setting in `SETTINGS` right?

Reply:
Ah, misunderstood you and thought you meant the last line of the hunk. I thought the `genkey_*` keys were in `SETTINGS`, but that was me confusing the long block of parser settings with the `SETTINGS` one...